FreeRTOS-Plus/Demo/FreeRTOS_Plus_FAT_TCP_CGI_RISC-V/Makefile

```diff
@@ -246,7 +246,7 @@ $(SDK_BUILD_DIR)/%.o : %.S
 $(OUT_ELF): $(OUT_OBJS) $(LINKER_SCRIPT) $(LINK_DEPS) libxil
 	@echo 'Building target: $@'
 	@echo '_______________________________'
-	$(GCC) $(CFLAGS) $(INCLUDES) $(OUT_OBJS) -o $@ $(LDFLAGS) $(LIBXIL_LDFLAGS) $(LDOPTS) -L$(LIBXIL_BUILD_DIR)
+	$(GCC) -v $(CFLAGS) $(INCLUDES) $(OUT_OBJS) -o $@ $(LDFLAGS) $(LIBXIL_LDFLAGS) $(LDOPTS) -L$(LIBXIL_BUILD_DIR)
```

Comment:
Do we need the -v?

Reply:
No.  Debuging change left in.  Will remove.